python/pyiceberg/avro/reader.py

```diff
@@ -103,6 +103,19 @@ def skip(self, decoder: BinaryDecoder) -> None:
         return None
 
 
+class DefaultReader(Reader):
+    default_value: Any
+
+    def __init__(self, default_value: Any) -> None:
+        self.default_value = default_value
+
+    def read(self, _: BinaryDecoder) -> Any:
+        return self.default_value
+
+    def skip(self, decoder: BinaryDecoder) -> None:
+        return None
```

Comment:
This isn't an issue, but I don't think that we need to return anything from `skip`, do we?

Reply:
Fair point, I've changed it into:
```suggestion
        pass
```